.github/workflows/ci-actions-incremental.yml

```diff
@@ -1184,7 +1184,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{ fromJson(needs.calculate-test-jobs.outputs.native_matrix) }}
-    runs-on: ${{matrix.os-name}}
+    runs-on: ubuntu-latest
```

Comment:
I'm surprised you have to start several jobs for it?

Reply:
You are right, we should be able to achieve the same with a single job. I will have a look at that in late August.

Reply:
I merged because it's still an improvement but let's try to reduce our CI usage if possible. Thanks!